PlaywrightDemoQA/WebTableTests.cs

```diff
@@ -15,12 +15,72 @@ public async Task DeleteRow()
             Headless = false,
         });
         var context = await browser.NewContextAsync();
-
         var page = await context.NewPageAsync();
         await page.GotoAsync("https://demoqa.com/webtables");
+        
         await page.Locator(".rt-tr-group")
             .Filter(new() { Has = page.GetByRole(AriaRole.Gridcell, new() { Name = "Alden", Exact = true }) })
             .GetByTitle("Delete").ClickAsync();
+       
         await Assertions.Expect(page.GetByText("Alden")).Not.ToBeVisibleAsync();
     }
-}
\ No newline at end of file
+
+    [Test]
+    public async Task EditRow()
+    {
+        using var playwright = await Playwright.CreateAsync();
+        await using var browser = await playwright.Chromium.LaunchAsync(new BrowserTypeLaunchOptions
+        {
+            Headless = false,
+        });
+        var context = await browser.NewContextAsync();
+
+        var page = await context.NewPageAsync();
+        await page.GotoAsync("https://demoqa.com/webtables");
+
+        await page.Locator(".rt-tr-group")
+            .Filter(new() { Has = page.GetByRole(AriaRole.Gridcell, new() { Name = "Alden", Exact = true }) })
+            .GetByTitle("Edit").ClickAsync();
+        await page.GetByPlaceholder("Last Name").FillAsync("Canter");
+        await page.GetByPlaceholder("Salary").FillAsync("13200");
```

Comment:
Task "increase by 10%" means that you need to read current value and increase it by 10%. Test should work for different current values but not for single current value

Reply:
Fixed, Thanks.